### ISSUE TITLE
Curator organizations interface

### ISSUE THREAD

**Issue by denis-yuen** [ga4gh/dockstore#2019]
### Desired behaviour

Placeholder for work pertaining to the curator GUI for the organizations feature.
Already present in wireframe in specification, will need GUI and the occasional webservice endpoint

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-240)
┆Issue Type: Story
┆Issue Number: DOCK-240
┆Epic: Organization for Dockstore

**Comment by agduncan94** [comment]
So the GUI has been implemented, though I am open for comments and improvements. Off the top of my head adding some sort of badge showing the count of requests or just that at least one request exists. This could be added to the dropdown on the top right of the page or only on the requests tab when on the accounts page. I could see this being displayed on the home page once we have personalized home pages implemented.
